packages/react-components/src/components/chart/hooks/useDataZoom.ts

```diff
@@ -1,11 +1,10 @@
-import { MutableRefObject, useEffect, useRef, useState } from 'react';
+import { MutableRefObject, useEffect, useRef } from 'react';
 import { useViewport } from '../../../hooks/useViewport';
 import { DataZoomComponentOption, EChartsType } from 'echarts';
 import { ECHARTS_GESTURE } from '../../../common/constants';
-import { DEFAULT_DATA_ZOOM, ECHARTS_ZOOM_DEBOUNCE_MS } from '../eChartsConstants';
```

Comment:
we can delete `ECHARTS_ZOOM_DEBOUNCE_MS` if its not used anywhere else

Reply:
Good call! I didn't check lol